src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3322,28 +3322,34 @@ internal EffectiveParameters(ImmutableArray<TypeWithAnnotations> types, Immutabl
             return paramRefKind;
         }
 
-        // In method group conversions, 'in' is allowed to match 'ref' and 'ref readonly' is allowed to match 'ref' or 'in'.
-        internal static bool AreRefsCompatibleForMethodConversion(RefKind x, RefKind y, CSharpCompilation compilation)
+        // In method group conversions,
+        // - 'in' is allowed to be converted to 'ref',
+        // - 'ref readonly' is allowed to be converted to 'ref' or 'in',
+        // - 'in' is allowed to be converted to 'ref readonly'.
```

Comment:
Reading through this chart, occurs to me that `out` should be allowed to convert to `ref`. I do **not** want to change the scope of the PR to include this. Just a thought that occured.

Reply:
Seems safe, but that probably needs a separate proposal, right?

Reply:
Agree. Not sure it's worth it, just occurred to me when reading this particular part of the change.